test/Test/Universum/Property.hs

```diff
@@ -0,0 +1,107 @@
+
+module Test.Universum.Property
+        ( hedgehogTestTree
+        ) where
+
+import Universum
+import Hedgehog
+import Test.Tasty
+import Test.Tasty.Hedgehog
+
+import qualified Hedgehog.Gen              as Gen
+import qualified Hedgehog.Range            as Range
+
+import qualified Data.ByteString           as B
+import qualified Data.ByteString.Lazy      as LB
```

Comment:
Are you using `stylish-haskell` for formatting imports? I think it would be better if you configure it _on-save_ for your editor 🙂 Also, `universum` already contains `.stylish-haskell.yaml` config.

Reply:
Ran stylish haskell, will set it up to run on save when I get spacemacs working well enough to switch over